packages/renderer/htmlUtils.ts

```diff
@@ -264,6 +267,13 @@ class HtmlUtils {
 
 				attrs = { ...attrs };
 
+				if (currentTag() === 'svg') {
+					isInsideSvg = true ;
```

Comment:
I'm not sure we can know that with an HTML parser (which is different from an SVG parser), so there's a chance that could be exploited to escape the sanitizer.